XLForm/XL/Descriptors/XLFormRowDescriptor.m

```diff
@@ -155,7 +156,9 @@ -(XLFormBaseCell *)cellForFormController:(XLFormViewController * __unused)formCo
         
         if ([bundle pathForResource:cellResource ofType:@"nib"]) {
             _cell = [[bundle loadNibNamed:cellResource owner:nil options:nil] firstObject];
-        } else {
+        }else if([bundleForCaller pathForResource:cellResource ofType:@"nib"]){
```

Comment:
Please respect the style guide: Add a space after `}` and before `{` and `(`.
Same thing below in `else`